syntax = "proto3";
package p1on;

import "color.proto";

//Color map converting scalars into colors
message ColorMap {
    //Color stops
    message Stop {
        //value at which to stop
        double value = 1;
        //Color at this stop
        Color color = 2;
    }

    //gradient stops
    repeated Stop stops = 1;
}
